pages/index.js

```diff
@@ -55,14 +55,18 @@ export default function Home() {
                 setActiveTab(key);
                 return;
             }
-            const wasm_bytes_response = lfortran_funcs.compile_code(sourceCode);
+            const wasm_bytes_response = lfortran_funcs.compile_code(sourceCode);            
             if (wasm_bytes_response) {
+                const start_compile = performance.now();
                 const [exit_code, ...compile_result] = wasm_bytes_response.split(",");
                 if (exit_code !== "0") {
                     setOutput(ansi_up.ansi_to_html(compile_result)); // print compile-time error found by lfortran to output
                 }
```

Comment:
I think we should print the compilation time to `stdout` in case of compilation error
```suggestion
                if (exit_code !== "0") {
                     // print compile-time error found by lfortran to output
                    setOutput(ansi_up.ansi_to_html(compile_result) + `\nCompilation Time: ${duration_compile} ms`);
                }
```